packages/core/package.json

```diff
@@ -68,5 +73,6 @@
     "storybook": "^7.0.2",
     "style-dictionary": "^3.7.2",
     "token-transformer": "0.0.32"
-  }
+  },
+  "gitHead": "b6bde2710dac0160e4df9e77ccd0ca0ca67d18e9"
```

Comment:
This is line is a temp publish artifact and shouldn't be committed 
I don't know why you have it - it should be automatically removed after publishing